### PR TITLE
Issue #9485: add example of AST for TokenType.LITERAL_NATIVE

### LINKED ISSUE #9485: update example of AST for TokenTypes.LITERAL_NATIVE
https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#LITERAL_NATIVE

needs to be extended to show example of AST.
Source to be updated: https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

PLEASE READ for more details: https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes

### PR DESCRIPTION
**Pull request by vivek286**
Closes #9485 

WhatsApp Image 2021-03-16 at 11 03 06 PM

Expected change in code
WhatsApp Image 2021-03-16 at 11 02 27 PM

### CODE REVIEW

**Comment by rdiachenko** [comment]
@vivek286 please update description according to https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-AST-in-TokenTypes (check PR example mentioned)

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1269,23 +1269,18 @@
...
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--POST_INC (++)
-     *                 |
-     *                 +--IDENT (x)
-     *         +--SEMI (;)
-     *         +--RCURLY (})
-     * +--RCURLY (})
+     * |--LITERAL_SYNCHRONIZED -&gt; synchronized
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1272]: I do not see "native" usage this example

**Review by romani — CHANGES_REQUESTED** [review]
item:

**Comment by romani** [comment]
if there will be no reply in a week, this PR be closed.

**Pull request closed without merging**